sql/plan/show_status.go

```diff
@@ -66,7 +68,28 @@ func (s *ShowStatus) Children() []sql.Node {
 
 // RowIter implements sql.Node interface.
 func (s *ShowStatus) RowIter(ctx *sql.Context, row sql.Row) (sql.RowIter, error) {
-	return sql.RowsToRowIter(), nil
+	var names []string
+	for name := range sql.SystemVariables.NewSessionMap() {
+		names = append(names, name)
+	}
+	sort.Strings(names)
+
+	var rows []sql.Row
+	for _, name := range names {
+		sysVar, val, ok := sql.SystemVariables.GetGlobal(name)
+		if !ok {
+			panic("attempted to retrieve non-existent system variable; shouldn't be possible")
```

Comment:
Better to just return an `error` here, even if we think this really can't be possible. It'll have a similar effect of failing the query, but it won't take down the entire sql-server for other sessions.